tests/e2e/hcp_cluster_test.go

```diff
@@ -260,4 +260,56 @@ var _ = Describe("HCP cluster testing",
 				textData = rosaClient.Parser.TextData.Input(out).Parse().Tip()
 				Expect(textData).To(ContainSubstring("Updated cluster '%s'", clusterID))
 			})
+
+		It("rosacli can show the details of HCP cluster well when describe - [id:54869]",
+			labels.Critical, labels.Runtime.Day2,
+			func() {
+				By("Get cluster description")
+				clusterDesc, err := clusterService.DescribeClusterAndReflect(clusterID)
+				Expect(err).To(BeNil())
+
+				By("Check values of description")
+				Expect(clusterDesc.Name).ToNot(BeEmpty())
+				Expect(clusterDesc.ID).ToNot(BeEmpty())
+				Expect(clusterDesc.ControlPlane).To(Equal("ROSA Service Hosted"))
+				Expect(clusterDesc.OpenshiftVersion).ToNot(BeEmpty())
+				Expect(clusterDesc.DNS).ToNot(BeEmpty())
+				Expect(clusterDesc.APIURL).ToNot(BeEmpty())
+				Expect(clusterDesc.Region).ToNot(BeEmpty())
+				Expect(clusterDesc.Availability[0]).To(HaveKeyWithValue("Control Plane", MatchRegexp("AZ")))
+				Expect(clusterDesc.Availability[1]).To(HaveKeyWithValue("Data Plane", MatchRegexp("AZ")))
+				Expect(clusterDesc.Nodes).To(HaveEach(HaveKey(MatchRegexp("Compute.*"))))
+				for _, v := range clusterDesc.Nodes {
+					for _, value := range v {
+						switch value.(type) {
+						case int:
+							Expect(value).To(BeNumerically(">=", 0))
+						case string:
+							Expect(value).To(MatchRegexp("^[0-9]+-[0-9]+$"))
+						default:
+							Expect(value).To(BeNil())
+						}
+					}
+				}
+				Expect(clusterDesc.Network[0]).To(HaveKey("Type"))
+				Expect(clusterDesc.Network[1]).To(HaveKey("Service CIDR"))
+				Expect(clusterDesc.Network[2]).To(HaveKey("Machine CIDR"))
+				Expect(clusterDesc.Network[3]).To(HaveKey("Pod CIDR"))
+				Expect(clusterDesc.Network[4]).To(HaveKey("Host Prefix"))
+				Expect(clusterDesc.Network[5]).To(HaveKeyWithValue("Subnets", MatchRegexp("^subnet-.{17}")))
```

Comment:
When it has multi subnets set it will fail.

Reply:
The value of subnets is string. Regexp is expecting only one subnet presents which should be present all the time. Even if multiple subnets are in string it will not fail